src/SignalR/clients/ts/signalr/src/LongPollingTransport.ts

```diff
@@ -95,33 +94,29 @@ export class LongPollingTransport implements ITransport {
         this._receiving = this._poll(this._url, pollOptions);
     }
 
-    private async _getAccessToken(): Promise<string | null> {
-        if (this._accessTokenFactory) {
-            return await this._accessTokenFactory();
-        }
-
-        return null;
-    }
-
-    private _updateHeaderToken(request: HttpRequest, token: string | null) {
+    private async _updateHeaderToken(request: HttpRequest): Promise<void> {
         if (!request.headers) {
             request.headers = {};
         }
-        if (token) {
-            request.headers[HeaderNames.Authorization] = `Bearer ${token}`;
-            return;
-        }
-        if (request.headers[HeaderNames.Authorization]) {
-            delete request.headers[HeaderNames.Authorization];
+        if (this._accessTokenFactory) {
+            const token = await this._accessTokenFactory();
+            if (token) {
```

Comment:
```suggestion
            if (token === undefined) {
```

Although, I hope no one is really using something falsy like `0` or `null` as their access token.

Reply:
I see this is old behavior and we want to delete the header in the `""` case. It might be slightly better to check for `""` or `undefined`, but it's clearly not a big deal.